library/Ivoz/Provider/Infrastructure/Persistence/Doctrine/ContactDoctrineRepository.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace Ivoz\Provider\Infrastructure\Persistence\Doctrine;
+
+use Doctrine\Bundle\DoctrineBundle\Repository\ServiceEntityRepository;
+use Ivoz\Provider\Domain\Model\Contact\Contact;
+use Ivoz\Provider\Domain\Model\Contact\ContactRepository;
+use Symfony\Bridge\Doctrine\RegistryInterface;
+
+/**
+ * ContactDoctrineRepository
+ *
+ * This class was generated by the Doctrine ORM. Add your own custom
+ * repository methods below.
+ */
```

Comment:
```suggestion
 * repository methods below.
 *
 * @template-extends ServiceEntityRepository<Contact>
 */
```

Reply:
Note that this file has been also copied (from LocationDoctrineRepository in this case), and no repository has that annotation. Do you want me to add it only to this Repository?

Reply:
This has been updated.